scripts/bootloader/generate_zip.py

```diff
@@ -67,6 +73,13 @@ def get_name(text_wrapper):
         merged.update(special_info[n])
         manifest['files'].append(merged)
 
+    if args.name is not None and update_type is not None:
+        manifest['name'] = '{}: {}'.format(args.name, update_type)
+    elif args.name is not None:
+        manifest['name'] = args.name
+    elif update_type is not None:
+        manifest['name'] = update_type
```

Comment:
Wouldn't this value be a duplicate of the "type" field?

Reply:
Yes.  The type field is used on the nRF Cloud backend, but not displayed to the user.  The name field is displayed to the user. In this example, the args.name was set to 'lte-gateway' and the type field was set to 'application'.  If this were instead a bootloader update, it would say 'lte-gateway: mcuboot'.  By combining these two strings, the user gets a better idea of the purpose of the bundle.
![image](https://user-images.githubusercontent.com/380268/134717442-a4b962ad-19d1-446d-855a-8de969b3389d.png)

Reply:
OK, I removed type from the name.